lib/utils/azure_queues.ts

```diff
@@ -3,11 +3,17 @@
  */
 import * as winston from "winston";
 
-import { IContext } from "azure-functions-types";
 import { QueueService } from "azure-storage";
 
 import { Option, some } from "fp-ts/lib/Option";
 
+export interface IQueueMessage extends QueueService.QueueMessageResult {
+  readonly id: string;
+  readonly nextVisibleTime: string;
+  readonly popReceipt: string;
+  readonly queueTrigger: string;
+}
+
 // Any delay must be less than 7 days (< 604800 seconds)
```

Comment:
would be nice to link to the relevant doc as this limit may change in the future

Reply:
actually this is an arbitrary value only set here. it *may* equal max(time to live), but not necessarily.